static/js/components/Main.jsx.template

```diff
@@ -82,6 +82,11 @@ const MainContent = ({ root }) => {
     store.dispatch(rotateLogoActions.rotateLogo());
   }, []);
 
+  let location = useLocation();
```

Comment:
Is `location` ever reassigned? If not use `const`

Reply:
Fixed.